nessie/jobs/generate_merged_student_feeds.py

```diff
@@ -48,14 +51,43 @@ class GenerateMergedStudentFeeds(BackgroundJob):
     staging_schema = destination_schema + '_staging'
     staging_schema_identifier = psycopg2.sql.Identifier(staging_schema)
 
-    def run(self, term_id=None):
-        """Loop through all records stored in the Calnet external schema and write merged student data to the internal student schema."""
-        app.logger.info(f'Starting merged profile generation job... (term_id={term_id})')
+    def run(self, term_id=None, backfill_new_students=False):
```

Comment:
I've locally tested the changes below to the best of my ability, but a run through our whole population seems like the only way to cover all the bases.